### DISCUSSION TITLE
版本号为什么有时带后缀有时却没有

### DISCUSSION THREAD

**Discussion by lxr760** [the1812/Bilibili-Evolved#4321] [提问 / Q&A] [answered]
家里2台电脑，都是chrome dev浏览器，版本Version 116.0.5845.32 (Official Build) dev (64-bit)

但插件的版本显示却不一样，有一台电脑显示是v2.7.4-50-g1965b5590     另外一台只有v2.7.4

![Snipaste_2023-07-16_12-21-13](https://github.com/the1812/Bilibili-Evolved/assets/8476457/cf8fccee-d82d-41ed-826c-72f986ad92ba)
![Snipaste_2023-07-16_12-21-21](https://github.com/the1812/Bilibili-Evolved/assets/8476457/7d353a2b-3695-4918-affb-fd4a677a5fc9)

**Comment by the1812** [comment, marked as answer]
上面的是 jsDelivr 源, 下面的是 GitHub 源

后缀表示距离这个版本 tag 的 commit 数量以及当前 commit hash, 0 的时候就不显示了
